scripts/cluster-install/rabbitmq_config.yaml

```diff
@@ -0,0 +1,22 @@
+rabbitmq:
+  username: cloudify
+  password: 'c10udify'
```

Comment:
It might be nice to generate the rabbit credentials on each install, since that'd be better practise (in case anyone from PS takes this to use in a customer environment).

Reply:
Great point, as I see it, we have two options of generating the rabbitmq credentials:  
1. Adding it to the config.yaml file. 
2. Writing a comment in the README that says 'you can modify the rabbitmq credentials in the rabbitmq_config.yaml'. 

Which one is preferable or is there another option I missed?

Reply:
I was thinking more like generating them randomly- a random string of 20 characters, for instance.

Reply:
Done.